ooni/oonicli.py

```diff
@@ -315,10 +320,10 @@ def post_director_start(_):
 
             net_test_loader.annotations = global_options['annotations']
 
-            director.startNetTest(net_test_loader,
-                                    global_options['reportfile'],
-                                    collector)
-        return director.allTestsDone
+            yield director.startNetTest(net_test_loader,
+                                        global_options['reportfile'],
+                                        collector,
+                                        global_options['no-yamloo'])
```

Comment:
So, this is basically a pair of coroutines? Like what you can do in python3 with `yield from`?